services/app-web/src/pages/LinkYourRates/LinkRateSelect.tsx

```diff
@@ -75,7 +77,7 @@ export const LinkRateSelect = ({
                 revision.formData.rateCertificationName ??
                 'Unknown rate certification',
             rateProgramIDs: programNames(
-                statePrograms,
```

Comment:
I still see the withdrawn rate in the link rate dropdown selection when filling out the rate details page on a new submission. Somewhere in this component should filter out rates with `withdrawInfo`. 

Also, should withdrawn rates be shown in the rates dropdown on the replace rate page?

Reply:
>Also, should withdrawn rates be shown in the rates dropdown on the replace rate page?

No they shouldn't. You can't use a withdrawn rate to replace a rate on some other random submission.

😮‍💨  I forgot I need to adjust the props for RateDetails still thank you for pointing out.

Reply:
@JasonLin0991 Oh wait your work is going to fix this  - since index rates with filter out withdrawn rates right

Reply:
Oh right ...eh,whatever. Yes, it can be done either way. I would probably make the API accept optional filters like I just added for state - where it either can return all states or return just a specific state. Same think for including withdrawn or not.

To speed this up for now, I'll just do it clientside on this PR... follow the existing pattern we have currently.